doc/source/concept_and_workflow/systemdeps.rst

```diff
@@ -0,0 +1,96 @@
+.. _systemdeps:
+
+Host Requirements To Build Images
+---------------------------------
+
+When building OS images, several tools and sub-systems are used
+and required on the host {kiwi} is called at. For example to
+build a virtual disk image tools to create a partition table
+and also tools to create filesystems needs to be present on the
+host that builds the image.
+
+The number of required components depends on the selected image
+type and the *features* used with the image. We cannot expect
+the users of {kiwi} to know about each and every component that
+is needed to build the image. Therefore a concept to help with
+the host requirements exists and is named `kiwi-systemdeps`
+
+The `kiwi-systemdeps` concept consists out of a collection of
+sub-packages provided with the `python3-kiwi` main package. Each
+individual package requires a number of tools and subsystem packages
+which belongs to the package category. There are the following
+systemdeps packages:
+
+kiwi-systemdeps-core:
```

Comment:
After reading the whole list, the information therein is very similar to each item. Another, alternative presentation could be useful to make it better readable and accessible. Maybe a table or a sublist?

For example, one presentation could be a sublist:

-----
`kiwi-systemdeps-X`:

  * **Purpose**: explains for what it is used for.
  * **Installs**: what it installs
  * **Dependencies**: what dependencies it needs
 -----

I prefer list over tables as tables are a nightmare on smartphones. :smile:  Especially if you have lots of columns.

It's just an idea, don't feel tempted to just revamp all the stuff. :slightly_smiling_face:

Reply:
I like the idea and adapted the docs as suggested :+1: